src/constants/upgradeConfig.ts

```diff
@@ -39,8 +41,30 @@ export const upgradeTokensList: {
 		| 'hasSushiApprove'
 		| 'hasIdleApprove'
 		| 'hasIbAlluoETHApprove'
+		| 'hasIbAlluoBTCApprove'
 		| 'hasIbAlluoUSDApprove';
```

Comment:
The upgrade flow for ibAlluoUSD doesnt work anymore, I just tried with a fresh wallet and I can't approve and upgrade, after approving, it doesnt enable the upgrade. I'm not sure where the issue is but it might have to do with the ordering of this and the ordering of the list of objects starting on line 47. 

Make a _new_ wallet and trasnfer it some ibAlluoUSD, try the approve + upgrade yourself.


<img width="1656" alt="Screen Shot 2022-09-13 at 5 04 51 PM" src="https://user-images.githubusercontent.com/5077880/190009106-39ffc685-4d18-4700-b7c8-d086b80a9996.png">

Reply:
I would also recommend just addressing one issue at a time, #790 probably works now, but since you also tried to ram in #780, I can't test #790 to see if it really works.